src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/jvm/JvmActionsAlarmMonitor.java

```diff
@@ -44,10 +44,12 @@ public JvmActionsAlarmMonitor(int dayBreachThreshold, int weekBreachThreshold, @
     Path dayMonitorPath = null;
     Path weekMonitorPath = null;
     if (persistencePath != null) {
-      String persistenceBase = persistencePath.getParent().toString();
-      String persistenceFile = persistencePath.getFileName().toString();
-      dayMonitorPath = Paths.get(persistenceBase, DAY_PREFIX + persistenceFile);
-      weekMonitorPath = Paths.get(persistenceBase, WEEK_PREFIX + persistenceFile);
+      Path persistenceBase = persistencePath.getParent();
```

Comment:
Please consider removing https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/pull/465/files#diff-218fa9f28b0f13a45438918579b436c2554f4f177ef344c422db01afb39e108bR73-R76